src/changelog/buildpacks/_posts/2023-12-18-java-openjdk-21.md

```diff
@@ -0,0 +1,7 @@
+---
+modified_at: 2023-12-19 12:00:00
```

Comment:
Filename is also wrong. I don't know if it has an impact on the deployment. Hence I can already approve the PR but dunno if the deployment will succeed.